config.js

```diff
@@ -53,10 +53,6 @@ module.exports = {
       'restify': path.join(__dirname, 'src/plugins/plugin-restify.js')
     },
 
-    // Valid entries are:
-    // 'express', 'hapi', 'http', 'restify'
-    excludedHooks: [],
```

Comment:
Good point. After an in person discussion, I'll put together a fix here: https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/414.